plugins/org.jkiss.dbeaver.ext.exasol/src/org/jkiss/dbeaver/ext/exasol/model/ExasolDataSource.java

```diff
@@ -137,7 +137,7 @@ public void initialize(@NotNull DBRProgressMonitor monitor)
 					"	s.SCHEMA_NAME as OBJECT_NAME," + 
 					"	s.SCHEMA_OWNER AS OWNER," + 
 					"CAST(NULL AS TIMESTAMP) AS created, " +
-					"	ADAPTER_SCRIPT," + 
+					"	" + (this.exasolCurrentUserPrivileges.getAtLeastV8() ? "'\"' || ADAPTER_SCRIPT_SCHEMA || '\".\"' || ADAPTER_SCRIPT_NAME || '\"' AS ADAPTER_SCRIPT" : "ADAPTER_SCRIPT") + "," +
```

Comment:
" " - what for it?
Also, Line is longer than 140 characters (found 222), and Line contains a tab character - this is for our codacy

Reply:
> " " - what for it?

To preserve the current formatting of the generated SQL query.

> Line is longer than 140 characters (found 222)

I'll try to split the line.

> Line contains a tab character - this is for our codacy

The added code is copy-paste, the file is already full of tab characters. Does it make sense to have alternating code style inside of a single file?